### ISSUE TITLE
Update gtk system requirement.

### ISSUE THREAD

**Issue by th3coop** [Komodo/komodo-documentation#1]
https://www.mozilla.org/en-US/firefox/35.0/system-requirements/ 
Says 2.18 and higher, not 2.10 and we use 2.18 in our builds.

**Comment by toddw-as** [comment]
The install.html file already say 2.18:
https://github.com/Komodo/komodo-documentation/blob/trunk/en-US/install.html

did someone not update docs.as.com? C.f.
http://docs.activestate.com/komodo/9.0/install.html#System_Req_Lin

**Comment by toddw-as** [comment]
Or maybe the original 9.0.0 GoldBits (RC3), didn't contain that change? As I updated GTK to 2.18 a week ago:
https://github.com/Komodo/komodo-documentation/commit/375acd3700fd0e4087f8f49c67d9a0ffede3bd5e#diff-c2fae9173ef9505ebc8ab4ea3def5950

**Comment by th3coop** [comment]
Right, we didn't think anything had been added since RC's started going out so I didn't update the docs.  That's what I get for not double checking.
